chronos/apps/http/httpserver.nim

```diff
@@ -845,10 +855,12 @@ proc acceptClientLoop(server: HttpServerRef) {.async.} =
     except TransportTooManyError:
       # Non critical error
       breakLoop = false
+    except TransportAbortedError:
+      # Non critical error
+      breakLoop = false
```

Comment:
There present some bugs in Nim's exception transformation inside of closure iterators (all async procedures are actually closure iterators). This why all this flags are present, it could be much more easier to use `finally` for example. But problem with loops, breaks and exceptions are present.